src/test/java/com/mturko/bestshop/ProductServiceTest.java

```diff
@@ -9,69 +9,61 @@
 
 import java.util.List;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
-@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class ProductServiceTest extends AbstractTest{
 
     Logger logger = LoggerFactory.getLogger(ProductServiceTest.class);
-    private static final String productInitialName = "testProduct";
-    private static final int productInitialPrice = 10;
-    private static final String productUpdatedName = "updatedTestProduct";
-    private static final int productUpdatedPrice = 20;
-    private static Product testProduct;
+    private final String productInitialName = "testProduct";
+    private final int productInitialPrice = 10;
+    private final int productUpdatedPrice = 20;
+    private Product expectedProduct;
+    private Product actualProduct;
 
     @Autowired
     ProductService productService;
 
-    @Test
-    void testSpringContextLoadForJunitTests() {
-        assertNotNull(productService);
-    }
-
-    @BeforeAll
-    static void init(){
-        Product product = new Product();
-        product.setName(productInitialName);
-        product.setPrice(productInitialPrice);
-        testProduct = product;
+    @BeforeEach
+    void setup() {
+        Product emptyProduct = productService.getProductByName(productInitialName);
+        Assertions.assertNull(emptyProduct);
```

Comment:
What is purpose of this assertion?

Reply:
to make sure we don't have this product created before tests

Reply:
Up to you :-)